swap_meet/item.py

```diff
@@ -1,2 +1,21 @@
+
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, condition=0, category=""):
+        self.category = category
+        self.condition = condition
+
+    def __str__(self):
+        return "Hello World!"
+
+    def condition_description(self):
```

Comment:
👍 

What would be returned if the condition were say -1, or how about 10.

Also, notice how repetitive all the conditions are. If we wanted to add additional conditions, we would need to write more code to handle them. What if we stored the conditions in a collection and could somehow calculate where in the collection the appropriate description for each condition could be found? If we do it carefully, adding additional conditions could be as simple as adding a new string to the collection, and there might be less code overall that would be easier to test (are ALL of these messages currently being tested?).